src/main/java/de/hpi/bpt/argos/eventHandling/EventPlatformRestEndpointImpl.java

```diff
@@ -116,13 +121,88 @@ protected void loadStatusUpdateEventType() {
 	 */
 	protected void loadDefaultEventType(File eventTypeFile) {
 		try {
+			DataFile file = entityManager.getDataFile(eventTypeFile.getAbsolutePath());
+
+			if (file.getModificationTimestamp() == eventTypeFile.lastModified()) {
+				logger.info(String.format("skipped default event type file '%1$s', since it was loaded already", eventTypeFile.getName()));
+				return;
+			}
+
 			String fileContent = new String(Files.readAllBytes(Paths.get(eventTypeFile.toURI())), StandardCharsets.UTF_8);
 
 			JsonObject jsonEventType = jsonParser.parse(fileContent).getAsJsonObject();
 			entityManager.createSimpleEventType(jsonEventType);
 
+			file.setModificationTimestamp(eventTypeFile.lastModified());
+			entityManager.updateEntity(file);
+
+		} catch (Exception e) {
+			logger.error("cannot load default event type from '" + eventTypeFile.getName() + "'.");
+            logger.trace("Reason: ", e);
+		}
+	}
+
+	/**
+	 * This method loads all backbone data files from the disk.
+	 */
+	protected void loadBackboneData() {
+		try {
+			PropertyEditor propertyEditor = new PropertyEditorImpl();
+
+			boolean loadBackboneData = Boolean.parseBoolean(propertyEditor.getProperty(Argos.getArgosBackendLoadBackboneDataPropertyKey()));
+
+			if (!loadBackboneData) {
+				logger.debug("loading backbone data skipped");
+				return;
+			}
+
+			String backboneDataDirectoryPath = propertyEditor.getProperty(Argos.getArgosBackendBackboneDataDirectoryPropertyKey());
```

Comment:
This is how we did it for all the other cases.
In this case, the property is not checked but just read just like when loading the default event types.